go.mod

```diff
@@ -113,5 +111,3 @@ require (
 replace github.com/Sirupsen/logrus => github.com/sirupsen/logrus v1.2.0
 
 replace github.com/influxdata/platform => /dev/null
-
-replace github.com/goreleaser/goreleaser => github.com/influxdata/goreleaser v0.97.0-influx
```

Comment:
This was being used so we could publish might snapshots to quay and s3 without creating a release.

Reply:
Is that needed, @BondAnthony ? I'm confused, because we also have this:

```yaml
release:
  disable: true
```

So I don't think we need `--snapshot` at all, unless I'm misunderstanding what is going on.

Reply:
I can revert the update of `goreleaser`, I'd just like to understand what's going on :)

Reply:
I think we needed this before splitting the goreleaser files. Let's move forward with your update.